src/objects/zcl_abapgit_object_iaxu.clas.abap

```diff
@@ -0,0 +1,216 @@
+CLASS zcl_abapgit_object_iaxu DEFINITION PUBLIC INHERITING FROM zcl_abapgit_objects_super FINAL.
+
+  PUBLIC SECTION.
+    INTERFACES zif_abapgit_object.
+
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+    DATA: mo_xml_api           TYPE REF TO cl_w3_api_xml3,
```

Comment:
suggest to remove mo_xml_api and always use a local variables instead as in `delete`.